content/millennium/r4/definitional-artifacts/questionnaire.md

```diff
@@ -31,7 +32,7 @@ The following fields are returned if valued:
         * [Required](https://hl7.org/fhir/R4/questionnaire-definitions.html#Questionnaire.item.required){:target="_blank"}
         * [Repeats](https://hl7.org/fhir/R4/questionnaire-definitions.html#Questionnaire.item.repeats){:target="_blank"}
         * [Max Length](https://hl7.org/fhir/R4/questionnaire-definitions.html#Questionnaire.item.maxLength){:target="_blank"}
-        * [Extension](https://hl7.org/fhir/R4/extension-maxdecimalplaces.html){:target="_blank"}
+        * [Extension](#extensions)
```

Comment:
Should i link to extensions section or the actual extension

Reply:
Do we list out which extensions we support somewhere?

Reply:
the extensions section should be fine.